acceptance/build_info_test.go

```diff
@@ -31,6 +31,7 @@ func TestBuildInfo(t *testing.T) {
 	l := localcluster.Create(1, stopper)
 	l.Start()
 	defer l.Stop()
+	defer assertNoEvents(t, l)
```

Comment:
certainly possible. Actually I'm going to see how much work it is to add the "awareness" in and do the filtering on `Stop()`.